plugin-build/src/main/kotlin/io/sentry/android/gradle/TracingInstrumentationExtension.kt

```diff
@@ -31,4 +31,32 @@ open class TracingInstrumentationExtension @Inject constructor(objects: ObjectFa
      */
     val forceInstrumentDependencies: Property<Boolean> = objects.property(Boolean::class.java)
         .convention(false)
+
+    /**
+     * Specifies a set of [InstrumentationFeature] features that are eligible for bytecode
+     * manipulation.
+     * Defaults to all available features of [InstrumentationFeature].
+     */
+    val features: SetProperty<InstrumentationFeature> =
+        objects.setProperty(InstrumentationFeature::class.java).convention(
+            setOf(
+                InstrumentationFeature.DATABASE,
+                InstrumentationFeature.FILE_IO
+            )
+        )
+}
+
+enum class InstrumentationFeature {
+    /**
+     * Does bytecode manipulation for 'androidx.sqlite' and 'androidx.room' libraries.
+     * It starts and finishes a Span within any CRUD operation performed by sqlite or room.
```

Comment:
`m`: I think we should focus on what this feature does rather than implementation details. The same applies to the comment for `FILE_IO`. Maybe something like:
```suggestion
     * When enabled the SDK will create spans for any CRUD operation performed by 'androidx.sqlite'
     * and 'androidx.room'. This feature uses bytecode manipulation.
```

Reply:
That looks better, thanks